src/components/Inspector.tsx

```diff
@@ -66,19 +66,22 @@ export const InspectorTabPanel = styled(TabPanel)`
     outline: none;
   }
 `
-
 export const InspectorTab = styled(Tab)`
   && {
+    font-family: inherit;
     background: none;
     padding: ${(props) => props.theme.grid.unit * 2}px;
     border-bottom-width: 1px;
```

Comment:
can we remove this since we added full border-bottom property on 77?

Reply:
done